network_importer/config.py

```diff
@@ -64,6 +64,9 @@ class NetboxSettings(BaseSettings):
     token: Optional[str]
     verify_ssl: bool = True
 
+    model_flag_tags: Optional[list] # List of tags that defines what objects to assign the model_flag_mode to.
```

Comment:
Might be better?

```python
model_flag_tags: List[str] = list()
```

That way you won't have to handle the case of `model_flag_tags == None` in `_is_tag_present`

Reply:
Thanks. Updated